commands.json

```diff
@@ -743,30 +789,34 @@
     "complexity": "O(N)",
     "history": [
       [
-          "2.0.0",
-          "Deprecated `WITHPAYLOADS` and `PAYLOAD` arguments"
+        "2.0.0",
+        "Deprecated `WITHPAYLOADS` and `PAYLOAD` arguments"
       ]
     ],
     "arguments": [
       {
         "name": "index",
-        "type": "string"
+        "type": "string",
+        "summary": "Specifies the name of the index. The index must be created using `FT.CREATE`."
       },
       {
         "name": "query",
-        "type": "string"
+        "type": "string",
+        "summary": "Specifies the query to profile and analyze performance."
       },
       {
         "name": "nocontent",
         "type": "pure-token",
         "token": "NOCONTENT",
-        "optional": true
+        "optional": true,
+        "summary": "Returns only the document IDs in the search results, excluding the content."
       },
       {
         "name": "verbatim",
         "type": "pure-token",
-        "token":"VERBATIM",
-        "optional": true
+        "token": "VERBATIM",
+        "optional": true,
+        "summary": "Searches using the exact query terms without stemming or synonym expansion."
```

Comment:
Align it with the command docs?
```suggestion
        "summary": "Searches using the exact query terms without stemming expansion."
```

Reply:
Looked at the docs, there it says:
![image](https://github.com/user-attachments/assets/521bd5fb-9c89-480a-b7fe-7e18272b805e)
Went with:
"Searches using the exact query terms without stemming or query expansion"